### PR TITLE
docs: add cached test result docs

### PR DESCRIPTION
**Pull request by povilasv**
## Pull request description 

add cached test result docs

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

**Review by haneabogdan — APPROVED** [review]

**Review by nicufk — APPROVED** [review]
Looks good, needs @jfermi review.

File: docs/docs/testkube-cloud/articles/cached-results.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Cached Test Results
+
+The Testkube cached test results allows you to see and inspect test execution results even when your Testkube agent is offline.
```
Comment (jfermi) [inline docs/docs/testkube-cloud/articles/cached-results.md:3]:
```suggestion
Testkube cached test results allows you to see and inspect test execution results even when your Testkube agent is offline.
```

File: docs/docs/testkube-cloud/articles/cached-results.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Overview
+
+![offline-main](../../img/offline-list.png)
+
+Testkube Cloud uses test execution data stored in Cloud to allow you inspect past test executions. This feature also works when your agent is online, but the Testkube agent doesn't have the test definition available in Kubernetes.
+
+Cached test results appear with read-only tag. These test cannot be updated. If you want to get rid of old test, you can go to Test Settings page and click "Delete Test".
```
Comment (jfermi) [inline docs/docs/testkube-cloud/articles/cached-results.md:11]:
```suggestion
Cached test results appear with a read-only tag. These tests cannot be updated. If you want to get rid of old tests, you can go to the Test Settings page and click "Delete Test".
```

**Review by jfermi — COMMENTED** [review]
A couple of editing suggestions.

**Pull request merged**